app/components/userAssignment/tempProjectLabel.tsx

```diff
@@ -0,0 +1,118 @@
+import { UPSERT_PROJECT } from "@/app/gqlQueries";
+import { AssignmentType } from "@/app/typeInterfaces";
+import { useUserDataContext } from "@/app/userDataContext";
+import { useMutation } from "@apollo/client";
+import { FormikValues,Formik } from "formik";
+import Image from "next/image";
+import { useParams } from "next/navigation";
+import React from "react";
+
+interface TempLabelProps {
+	assignment: AssignmentType;
+	tempProjectOpen: boolean;
+	setTempProjectOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const TempProjectLabel = ({
+	assignment,
+	tempProjectOpen,
+	setTempProjectOpen,
+}: TempLabelProps) => {
+	const {userId} = useParams();
+	console.log(assignment, "ASSIGNEMNT IN TEMP");
```

Comment:
Should we remove this and other `console.log` statements?

Reply:
Absolutely will remove these